react-frontend/src/components/NumberInput/NumberInput.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import './NumberInput.css'
+
+const NumberInput = ({ label, value, onChange }) => {
+  return (
+    <div className="number-input">
+      <label>{label}</label>
+      <input type="number" value={value} onChange={onChange} />
+    </div>
```

Comment:
```js
<label className="number-input">
      <span>{label}</span>
      <input type="number" value={value} onChange={onChange} />
 </label>
```

You can use something like that to make it accessible. Otherwise you need to use https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label `label + for` attribute to connect the label to the input

Reply:
Thanks, refactored in [0c2832f](https://github.com/YaroslavKSE/AuctionWeb/pull/2/commits/0c2832f95bc0ca4eb2cb355a368e619b1378eb2b)